src/main/resources/assets/enderio/textures/block/tesseract_portal.png.mcmeta

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please rename this file given we aren't calling it "tesseract". Just go with "vacuum_portal" or similar